packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -65,8 +65,9 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
     );
   }
 
-  public get selectedKey(): string | undefined {
-    return this.state.selectedKey;
+  public get selectedKey(): string | null | undefined {
```

Comment:
Since this is a pseudo-internal method, can we just add:
```ts
public get getSelectedKeySpecifed(): boolean {
  return 'selectedKey' in this.props;
}
```
And then logic can use that for consideration along with `selectedKey`?

Reply:
It's not pseudo-internal, this is part of the public interface.